packages/components/buttons/primary-button/src/primary-button.styles.ts

```diff
@@ -34,7 +34,7 @@ const getButtonStyles = (
     align-items: center;
     color: ${designTokens.colorSurface};
     transition: background-color ${designTokens.transitionLinear80Ms};
-    padding: ${designTokens.paddingForButton};
+    padding: 0 var(--spacing-30);
```

Comment:
```suggestion
    padding: 0 ${designTokens.spacing30};
```

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/098974e6596998284f5d665ce8a0a416310509e0
Thank you